widgetssdk/src/main/java/com/glia/widgets/filepreview/ui/FilePreviewActivity.java

```diff
@@ -55,7 +56,9 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void handleStatusBarColor() {
-        getWindow().setStatusBarColor(ContextCompat.getColor(this, this.theme.getBrandPrimaryColor()));
+        if (theme != null && theme.getBrandPrimaryColor() != null) {
+            getWindow().setStatusBarColor(ContextCompat.getColor(this, theme.getBrandPrimaryColor()));
```

Comment:
I added an extentionFunction in ViewExtentions

View.changeStatusBarColor(color: Int?)

Reply:
I think that’s on another branch, and also this is a Java file.